packages/opentelemetry-core/src/platform/node/index.ts

```diff
@@ -17,7 +17,8 @@
 export * from './BasePlugin';
 export * from './environment';
 export * from './hex-to-base64';
-export * from './id';
+export * from './RandomIdGenerator';
```

Comment:
Since I am new to JS, it may be a dumb question: 
Do you mean when we write code under .../node repo and want to export it, we should also add an `export * from '../../opentelemetry-core/src/platform/node/RandonIdGenerator` in the "opentelemetry-web" index file to solve dependency problem and pass through CircleCI? 
Or you mean I should add code under "opentelemetry-core/src/platform/browser"?
I just read through the code under opentelemetry-web, and still not sure what to do.

Reply:
When the project is compiled for web, the `node` folder is completely replaced by the `web` folder. This means if you do `import { MyClass } from './platform/node/myclass';`, in the web this will be transparently replaced with `import { MyClass } from './platform/web/myclass';`. So the exports from node and web need to have identical interfaces.

Reply:
You should not export anything from `platform/node` directly, all you have to do is to add a new stuff into corresponding `index.ts` in folder `platform/node/` and `platform/browser`. Platform itself is exported already so anything you add in those indexes will be exported.

Reply:
Thank you for you instructions, I have already get this done!